data_structures/src/chain.rs

```diff
@@ -635,6 +636,76 @@ impl SuperBlock {
         ]
         .concat()
     }
+
+    /// Generate a Proof of Inclusion for a given Superblock tally root, provided the DR transaction
+    /// and the blocks that the superblock contains
+    /// If the DR transaction is not found within those blocks, returns None
+    pub fn dr_proof_of_inclusion(
+        &self,
+        blocks: &[Block],
+        tx: DRTransaction,
```

Comment:
This should be a reference, it will save some clones from the tests.

```suggestion
        tx: &DRTransaction,
```